app/models/user.rb

```diff
@@ -8,4 +8,7 @@ class User < ApplicationRecord
                     uniqueness: { case_sensitive: false }
   has_secure_password
   validates :password, presence: true, length: { minimum: 6 }
+
+  has_many :questions
+  has_many :answers
```

Comment:
It's good that there's a `has_many` relationship here, but it doesn't seem to be reflected in my database. I would expect there to be a table like `users_and_questions` and `users_and_answers`, but I don't see anything like that. Is your database any different? These are all the tables I have.

```
qna_dev=# \d
                 List of relations
 Schema |          Name          |   Type   | Owner 
--------+------------------------+----------+-------
 public | answers                | table    | allen
 public | answers_id_seq         | sequence | allen
 public | ar_internal_metadata   | table    | allen
 public | lectures               | table    | allen
 public | lectures_id_seq        | sequence | allen
 public | lecturetaggings        | table    | allen
 public | lecturetaggings_id_seq | sequence | allen
 public | questions              | table    | allen
 public | questions_id_seq       | sequence | allen
 public | schema_migrations      | table    | allen
 public | topics                 | table    | allen
 public | topics_id_seq          | sequence | allen
 public | topictaggings          | table    | allen
 public | topictaggings_id_seq   | sequence | allen
 public | users                  | table    | allen
 public | users_id_seq           | sequence | allen
(16 rows)

qna_dev=# \d answers
                                      Table "public.answers"
   Column    |            Type             |                      Modifiers                       
-------------+-----------------------------+------------------------------------------------------
 id          | integer                     | not null default nextval('answers_id_seq'::regclass)
 correct     | boolean                     | 
 content     | character varying           | 
 question_id | integer                     | 
 created_at  | timestamp without time zone | not null
 updated_at  | timestamp without time zone | not null
 user_id     | integer                     | 
Indexes:
    "answers_pkey" PRIMARY KEY, btree (id)

qna_dev=# \d questions
                                      Table "public.questions"
    Column    |            Type             |                       Modifiers                        
--------------+-----------------------------+--------------------------------------------------------
 id           | integer                     | not null default nextval('questions_id_seq'::regclass)
 user_id      | integer                     | 
 created_at   | timestamp without time zone | not null
 updated_at   | timestamp without time zone | not null
 content      | character varying           | 
 published_at | timestamp without time zone | 
Indexes:
    "questions_pkey" PRIMARY KEY, btree (id)
Referenced by:
    TABLE "topictaggings" CONSTRAINT "fk_rails_8e3cf8ccb6" FOREIGN KEY (question_id) REFERENCES questions(id)
    TABLE "lecturetaggings" CONSTRAINT "fk_rails_bc96817846" FOREIGN KEY (question_id) REFERENCES questions(id)

qna_dev=# \d users
                                        Table "public.users"
     Column      |            Type             |                     Modifiers                      
-----------------+-----------------------------+----------------------------------------------------
 id              | integer                     | not null default nextval('users_id_seq'::regclass)
 username        | character varying           | 
 email           | character varying           | 
 created_at      | timestamp without time zone | not null
 updated_at      | timestamp without time zone | not null
 password_digest | character varying           | 
 role            | character varying           | 
Indexes:
    "users_pkey" PRIMARY KEY, btree (id)
    "index_users_on_email" UNIQUE, btree (email)
    "index_users_on_username" UNIQUE, btree (username)
```

I've run migrations, but nothing happened when I did.